packages/core/src/interfaces/axis-scales.ts

```diff
@@ -56,6 +56,45 @@ export interface AxisOptions {
 	};
 }
 
+/**
+ * customize time series scales
+ */
+export interface TimeScaleOptions {
+	addSpaceOnEdges?: number;
+	/**
+	 * if it's true, days are shown as mon-sun,
+	 * otherwise days are shown as number 1-31
+	 */
+	showDayName?: boolean;
+	/**
+	 * formats for each time interval
+	 */
+	timeIntervalFormats?: TimeIntervalFormats;
+	/**
+	 * local code, ie. "en-US"
```

Comment:
can we add a link here to the possible locale codes please? Maybe an upkg link to the locales folder in `fns` or if they have a page with the list that'd be great